app/assets/javascripts/initializers/initializeTimeFixer.js

```diff
@@ -1,71 +1,61 @@
-function initializeTimeFixer() {
-  var utcTime = document.getElementsByClassName('utc-time');
-  var utcDate = document.getElementsByClassName('utc-date');
-  var utc = document.getElementsByClassName('utc');
+'use strict';
 
-  if (!utc) {
-    return;
-  }
+/* eslint-disable no-param-reassign */
```

Comment:
Could you remove this and add `    /* eslint-disable-next-line no-param-reassign */` before line 43?

Reply:
Sure